### PR TITLE
Added technical note related to Ansible version

### PR DESCRIPTION
**Pull request by ahardin-rh**
Added per discussion with @brenton

### CODE REVIEW

**Comment by brenton** [comment]
:+1:

File: release_notes/ocp_3_7_release_notes.adoc [hunk 1]
```diff
@@ -1377,6 +1377,16 @@ explicitly setting `env name` to  `key mappings`.
 
 {product-title} 3.7 introduces the following notable technical changes.
 
+[discrete]
+[[use-ansible-version-that-ships-with-OCP]]
+=== Use the Ansible Version Shipping with {product-title}
+
+Use the version of Ansible that ships in the {product-title} channels (version
+2.3) instead of the version that ships with Red Hat Enterprise Linux (RHEL).
```
Comment (kalexand-rh) [inline release_notes/ocp_3_7_release_notes.adoc]: I'd just say RHEL. It's defined earlier in the topic.

File: release_notes/ocp_3_7_release_notes.adoc [hunk 2]
```diff
@@ -1377,6 +1377,16 @@ explicitly setting `env name` to  `key mappings`.
 
 {product-title} 3.7 introduces the following notable technical changes.
 
+[discrete]
+[[use-ansible-version-that-ships-with-OCP]]
+=== Use the Ansible Version Shipping with {product-title}
+
+Use the version of Ansible that ships in the {product-title} channels (version
```
Comment (kalexand-rh) [inline release_notes/ocp_3_7_release_notes.adoc]: s/ (version 2.3)/, version 2.3,

File: release_notes/ocp_3_7_release_notes.adoc [hunk 3]
```diff
@@ -1377,6 +1377,16 @@ explicitly setting `env name` to  `key mappings`.
...
+2.3) instead of the version that ships with Red Hat Enterprise Linux (RHEL).
+Later versions of {product-title} use the version from RHEL. However, on
+{product-title} 3.7 if you use Ansible 2.4, you can experience
```
Comment (kalexand-rh) [inline release_notes/ocp_3_7_release_notes.adoc]: s/3.7/3.7,

File: release_notes/ocp_3_7_release_notes.adoc [hunk 4]
```diff
@@ -1377,6 +1377,16 @@ explicitly setting `env name` to  `key mappings`.
...
+link:https://bugzilla.redhat.com/show_bug.cgi?id=1575063[*BZ#1575063*].
```
Comment (kalexand-rh) [inline release_notes/ocp_3_7_release_notes.adoc]: I'd add "the memory allocation error described in" before the bug link.

**Review by kalexand-rh — COMMENTED** [review]
Couple of suggestions. I might also add a link with information about downgrading Ansible.

**Comment by brenton** [comment]
For downgrading, disabling the Ansible 2.4 channel and running `yum downgrade ansible-2.3*` would do the trick.

**Comment by ahardin-rh** [comment]
Updated. Thanks!

**Pull request merged**